content/docs/getting-started/installation/kubernetes/powerstore/csmoperator/_index.md

```diff
@@ -98,6 +98,9 @@ To deploy the Operator, follow the instructions available [here](../../../operat
 | X_CSI_POWERSTORE_NODE_NAME_PREFIX | Prefix to add to each node registered by the CSI driver | Yes | "csi-node"
 | X_CSI_FC_PORTS_FILTER_FILE_PATH | To set path to the file which provides a list of WWPN which should be used by the driver for FC connection on this node | No | "/etc/fc-ports-filter" |
 | GOPOWERSTORE_DEBUG | Enable/Disable gopowerstore library-level debugging. | No | false |
+| X_CSI_NFS_CLIENT_PORT      | Define port for the HBNFS client. | No | "2050" |
```

Comment:
I think it would be nice to mention Host based NFS instead of HBNFS in our docs.

Reply:
We should not be using the acronym at all in our documentation. Thanks!